### ISSUE TITLE
NPE on app bootup

### ISSUE THREAD

**Issue by greenlaw110** [actframework/act-storage#1]
error stack:

```
Exception in thread "main" java.lang.NullPointerException
	at act.storage.EntityClassEnhancer.visit(EntityClassEnhancer.java:45)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.cli.bytecode.CommanderEnhancer.visit(CommanderEnhancer.java:60)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.util.SimpleBean$ByteCodeEnhancer.visit(SimpleBean.java:242)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.mail.bytecode.MailerEnhancer.visit(MailerEnhancer.java:70)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.controller.bytecode.ControllerEnhancer.visit(ControllerEnhancer.java:63)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.util.SingletonEnhancer.visit(SingletonEnhancer.java:54)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.util.DataObjectEnhancer.visit(DataObjectEnhancer.java:58)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.db.EntityClassEnhancer.visit(EntityClassEnhancer.java:56)
	at act.asm.ClassVisitor.visit(ClassVisitor.java:113)
	at act.db.ebean2.ModelBaseEnhancer.visit(ModelBaseEnhancer.java:28)
	at act.asm.ClassReader.accept(ClassReader.java:622)
	at act.asm.ClassReader.accept(ClassReader.java:506)
	at act.app.AppClassLoader.asmEnhance(AppClassLoader.java:477)
	at act.app.AppClassLoader.enhance(AppClassLoader.java:464)
	at act.app.AppClassLoader.enhancedBytecode(AppClassLoader.java:510)
	at act.app.AppCompiler$3.findType(AppCompiler.java:204)
	at act.app.AppCompiler$3.findType(AppCompiler.java:148)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.askForType(LookupEnvironment.java:151)
	at org.eclipse.jdt.internal.compiler.lookup.UnresolvedReferenceBinding.resolve(UnresolvedReferenceBinding.java:101)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.resolveType(BinaryTypeBinding.java:212)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.getTypeFromTypeSignature(LookupEnvironment.java:1553)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.createMethod(BinaryTypeBinding.java:788)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.createMethods(BinaryTypeBinding.java:912)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.cachePartsFrom(BinaryTypeBinding.java:524)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.createBinaryTypeFrom(LookupEnvironment.java:722)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.createBinaryTypeFrom(LookupEnvironment.java:705)
	at org.eclipse.jdt.internal.compiler.Compiler.accept(Compiler.java:301)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.askForType(LookupEnvironment.java:156)
	at org.eclipse.jdt.internal.compiler.lookup.UnresolvedReferenceBinding.resolve(UnresolvedReferenceBinding.java:101)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.resolveType(BinaryTypeBinding.java:212)
	at org.eclipse.jdt.internal.compiler.lookup.BinaryTypeBinding.superclass(BinaryTypeBinding.java:1875)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.detectHierarchyCycle(ClassScope.java:1240)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.detectHierarchyCycle(ClassScope.java:1195)
	at org.eclipse.jdt.internal.compiler.ast.QualifiedTypeReference.getTypeBinding(QualifiedTypeReference.java:123)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.internalResolveType(TypeReference.java:495)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveType(TypeReference.java:594)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveType(TypeReference.java:590)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveSuperType(TypeReference.java:564)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.findSupertype(ClassScope.java:1320)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.connectSuperclass(ClassScope.java:952)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.connectTypeHierarchyWithoutMembers(ClassScope.java:1171)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.detectHierarchyCycle(ClassScope.java:1306)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.detectHierarchyCycle(ClassScope.java:1195)
	at org.eclipse.jdt.internal.compiler.ast.SingleTypeReference.getTypeBinding(SingleTypeReference.java:67)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.internalResolveType(TypeReference.java:495)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveType(TypeReference.java:594)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveType(TypeReference.java:590)
	at org.eclipse.jdt.internal.compiler.ast.TypeReference.resolveSuperType(TypeReference.java:564)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.findSupertype(ClassScope.java:1320)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.connectSuperclass(ClassScope.java:952)
	at org.eclipse.jdt.internal.compiler.lookup.ClassScope.connectTypeHierarchy(ClassScope.java:1113)
	at org.eclipse.jdt.internal.compiler.lookup.CompilationUnitScope.connectTypeHierarchy(CompilationUnitScope.java:324)
	at org.eclipse.jdt.internal.compiler.lookup.LookupEnvironment.completeTypeBindings(LookupEnvironment.java:249)
	at org.eclipse.jdt.internal.compiler.Compiler.internalBeginToCompile(Compiler.java:843)
	at org.eclipse.jdt.internal.compiler.Compiler.beginToCompile(Compiler.java:385)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:431)
	at org.eclipse.jdt.internal.compiler.Compiler.compile(Compiler.java:417)
	at act.app.AppCompiler.compile(AppCompiler.java:116)
	at act.app.DevModeClassLoader.compileSources(DevModeClassLoader.java:160)
	at act.app.DevModeClassLoader.scan(DevModeClassLoader.java:92)
	at act.app.App.scanAppCodes(App.java:1209)
	at act.app.App.refresh(App.java:552)
	at act.app.AppManager.load(AppManager.java:83)
	at act.app.AppManager$_F$1.visit(AppManager.java:155)
	at act.app.AppManager$_F$1.visit(AppManager.java:151)
	at org.osgl.Osgl$Visitor.apply(Osgl.java:2226)
	at org.osgl.Osgl$Visitor.apply(Osgl.java:2216)
	at act.app.AppScanner.scan(AppScanner.java:94)
	at act.app.AppScanner.scan(AppScanner.java:74)
	at act.app.AppManager.loadSingleApp(AppManager.java:61)
	at act.Act.start(Act.java:319)
	at act.Act.startApp(Act.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at act.boot.app.RunApp.start(RunApp.java:84)
	at act.boot.app.RunApp.start(RunApp.java:62)
	at act.Act.start(Act.java:640)
	at com.yabonza.Yabonza.main(Yabonza.java:19)
```

**Comment by greenlaw110** [comment]
The issue is found on act-storage 0.10.0

**Comment by greenlaw110** [comment]
fixed in 0.10.1
